offchain/rollups-events/tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+certs/*
+!certs/*.sh
```

Comment:
The script file could live outside the certs dir, so we don't mix temporary files with committed ones.